doc/adr/0009-user-interface-look-and-feel.md

```diff
@@ -0,0 +1,30 @@
+# 9. User interface look and feel
+
+Date: 2018-07-18
+
+## Status
+
+Proposed
+
+## Context
+
+We are building a new digital service for CCS.
+
+[CCS-ADR-0002][ccs-adr-0002] says that all new services should have a common 
+user experience, based on the GOV.UK Design Patterns and Frontend Toolkit.
+
+These patterns should be amended to use the CCS colours and brand, but should
+remain consistent with GOV.UK where possible.
+
+## Decision
+
+We will follow this CCS Architectural Decision and use the GOV.UK Frontend
```

Comment:
Should we reword this to say "GOV.UK Design System", since this has now superseded the frontend toolkit?